index.js

```diff
@@ -6,15 +6,41 @@ class QuickLRU {
 			throw new TypeError('`maxSize` must be a number greater than 0');
 		}
 
+		if (typeof options.maxAge === 'number' && options.maxAge === 0) {
+			throw new TypeError('`maxAge` must be a number greater than 0');
+		}
+
 		this.maxSize = options.maxSize;
+		this.maxAge = options.maxAge || 0;
 		this.onEviction = options.onEviction;
 		this.cache = new Map();
 		this.oldCache = new Map();
 		this._size = 0;
 	}
 
-	_set(key, value) {
-		this.cache.set(key, value);
+	_hasExpired(key, item) {
```

Comment:
`_hasExpired` makes it sound like it returns a boolean.

Reply:
This new name makes more sense for you, or maybe should we split the function in order to be more atomic?